Sources/Framework/Views/IconTextField.swift

```diff
@@ -68,8 +63,8 @@ class IconTextField: UIView {
     private func setUpLayout() {
         verticalStack.edgeAnchors == edgeAnchors
 
-        iconView.setContentHuggingPriority(UILayoutPriorityRequired, for: .horizontal)
-        textField.setContentHuggingPriority(UILayoutPriorityDefaultLow, for: .horizontal)
+        iconView.setContentHuggingPriority(UILayoutPriority.required, for: .horizontal)
+        textField.setContentHuggingPriority(UILayoutPriority.defaultLow, for: .horizontal)
```

Comment:
It could be shorter, you don't need to specify `UILayoutPriority` struct name here:
```swift
textField.setContentHuggingPriority(.defaultLow, for: .horizontal)
```